test/kubemark/start-kubemark.sh

```diff
@@ -402,21 +446,31 @@ fi
 
 # master machine name format: {KUBE_GCE_ZONE}-kubemark-tp-1-master
 # destination file " --resource-providers=/etc/srv/kubernetes/kube-scheduler/rp-kubeconfig"
+# TODO: do the same for kube-controller-manager for each TP as well
+#
 if [[ "${SCALEOUT_CLUSTER:-false}" == "true" ]]; then
```

Comment:
this "if" code should be merge with previous "if" code, and also for loop in this "if" code should be merged with previous one.

Reply:
did a mini refactor to make the code path a bit clean as a minimum effort in this PR.